lib/deaf_grandma.rb

```diff
@@ -21,7 +21,17 @@ def run!
   def speak(input)
 
     #Implement your code here <<<<<<<<<
-
+    if input != input.upcase
+      "SPEAK UP SONNY!"
```

Comment:
This is totally optional but i think pulling out these hard coded responses into their own constants (or inside an Array#freeze) would make this much easier to modify if you had to maintain it in the future.